en_ewt-ud-test.conllu

```diff
@@ -7865,7 +7865,7 @@
 8	Street	Street	PROPN	NNP	Number=Sing	1	list	1:list	SpaceAfter=No
 9	,	,	PUNCT	,	_	8	punct	8:punct	_
 10	EB	EB	PROPN	NNP	Number=Sing	8	appos	8:appos	SpaceAfter=No
-11	3801a	3801a	NUM	CD	NumType=Card	10	nummod	10:nummod	_
+11	3801a	3801a	NUM	CD	NumForm=Word|NumType=Card	10	nummod	10:nummod	_
```

Comment:
Hmm...should this be `NumForm=Combi`?

Reply:
I guess the issue is we have identifiers (for an entity such as an apartment) that can be letters, numbers, or both. So, in a street address, what is the right way to handle:

- Apt. **1** - NUM, NumForm=Digit, NumType=Card
- Apt. **A** - PROPN? no numeric features because no digits involved?
- Apt. **1A** - NUM? NumForm=Combi because it contains both letters and digits, or Digit because there is no spelled-out word or ordinal suffix?

Reply:
Maybe we should leave that as a separate issue then?

Reply:
You mean drop the `NumForm` for now until it is resolved? I'd be OK with that

Reply:
Dropped it for now.  In the train set, it is treated as the head of a `Number=Sing` construction